app/src/main/java/org/p2p/wallet/striga/sms/onramp/StrigaOnRampSmsStorage.kt

```diff
@@ -0,0 +1,21 @@
+package org.p2p.wallet.striga.sms.onramp
+
+import org.p2p.wallet.striga.sms.StrigaSmsStorageContract
+import org.p2p.wallet.striga.user.StrigaStorageContract
+
+class StrigaOnRampSmsStorage(
```

Comment:
do we really need a proxy class for these fields? 
maybe we can put them inside StrigaStorageContract and leave only two of them? 
i think logic is same for them

Reply:
You're right. I just realized, that it actually doesn't make sense to even have 2 different timers for both screens, because they don't overlap each other at all, we can use the same fields for different blocking timers.